11_webui.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+set -eux
+source common.sh
+
+figlet "Deploying web ui" | lolcat
+eval "$(go env)"
+
+
+export UIPATH="$GOPATH/src/github.com/kubevirt/web-ui-operator"
+
+oc new-project kubevirt-web-ui
+cd $UIPATH/deploy
+oc apply -f service_account.yaml
+oc adm policy add-scc-to-user anyuid -z kubevirt-web-ui-operator
```

Comment:
This is not needed anymore - since kubevirt-web-ui-operator:v0.0.9 (built today)